partner_capital/models/res_partner.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2015 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# Copyright 2015 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    capital_country_id = fields.Many2one(
+        'res.country',
+        "Capital country",
+        help="Country of origin of this company's capital.")
+    capital_amount = fields.Float(
```

Comment:
You can make this now into a Monetary field with keyword currency_field='capital_currency_id' to automagically display this field everywhere as a money amount, together with the right currency.